src/userConfig_sample.h

```diff
@@ -49,6 +49,7 @@ enum MACHINE {
 #define PASS "CleverCoffee"        // default password for WiFiManager
 #define MAXWIFIRECONNECTS 5        // maximum number of reconnection attempts, use -1 to deactivate
 #define WIFICONNECTIONDELAY 10000  // delay between reconnects in ms
+#define NOINTERNET 1               // Use libraries for the webinterface from the internet (0) or from the local filesystem (1). 0 has slightly faster load times
```

Comment:
We have CONNECTMODE = 0 (offline). What is the different to NOINTERNET?

Reply:
The difference is that CONNECTMODE == 0 does not attempt to connect to Wifi while this is just not loading any dependencies for the website from the internet. Maybe we can name them differently to distinguish better

Reply:
I think we should remove this line and make it the default as we discussed.

Reply:
If we're sure that most people won't need the previous behaviour then sure. We can just allow changing it in the code for those rare cases